src/tests/database/settingsDB.test.ts

```diff
@@ -0,0 +1,49 @@
+import SettingsModel, {defaultSettings} from '../../database/models/SettingsModel';
+import {getSettings} from '../../database/settingsDB';
+import mongoose from 'mongoose';
+
+describe('Settings DB', () => {
+  beforeAll(async () => {
+    await mongoose.connect(process.env.MONGO_URL + 'settings' as string, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+      useFindAndModify: false,
+    });
+  });
+
+  afterAll(async () => {
+    await mongoose.connection.db.dropDatabase();
+    await mongoose.connection.close();
+  });
+
+  test('Settings Already Exist', async () => {
+    const guildId = 'randomString';
+    const doc = defaultSettings(guildId);
+    await SettingsModel.create(doc);
+    const result = await getSettings(guildId);
+    expect(result._id).toBe(doc._id);
+    expect(result.prefix).toBe(doc.prefix);
+    expect(result.timezone).toBe(doc.timezone);
+    expect(result.pinsEnabled).toBe(doc.pinsEnabled);
+    expect(result.quotesEnabled).toBe(doc.quotesEnabled);
+    expect(result.verificationEnabled).toBe(doc.verificationEnabled);
+    expect(result.birthdayAnnouncementsEnabled).toBe(doc.birthdayAnnouncementsEnabled);
+    expect(result.morningAnnouncementsEnabled).toBe(doc.morningAnnouncementsEnabled);
+    expect(result.dueDatesEnabled).toBe(doc.dueDatesEnabled);
+  });
+
+  test('Settings Dont Exist', async () => {
+    const guildId = 'randomString';
+    const doc = defaultSettings(guildId);
+    const result = await getSettings(guildId);
+    expect(result._id).toBe(doc._id);
+    expect(result.prefix).toBe(doc.prefix);
+    expect(result.timezone).toBe(doc.timezone);
+    expect(result.pinsEnabled).toBe(doc.pinsEnabled);
+    expect(result.quotesEnabled).toBe(doc.quotesEnabled);
+    expect(result.verificationEnabled).toBe(doc.verificationEnabled);
+    expect(result.birthdayAnnouncementsEnabled).toBe(doc.birthdayAnnouncementsEnabled);
+    expect(result.morningAnnouncementsEnabled).toBe(doc.morningAnnouncementsEnabled);
+    expect(result.dueDatesEnabled).toBe(doc.dueDatesEnabled);
+  });
+});
```

Comment:
Can you add a test to confirm that changing a setting actually does so in the DB?

This is assuming you've actually made a fake DB instance in the test and not a mock :P

Reply:
It's not even a fake DB instance. It's a real DB instance that's just stored in memory.

Also, I can't add a test to confirm changing settings because I haven't implemented the ability to change settings yet.

Reply:
This PR just adds the settings command - the settings command displays the guilds settings, changing them requires more implementation as some settings are booleans, some are strings, etc. so it is a more complicated command/commands.

Reply:
Sounds good, will approve